forgerock-auth/src/main/java/org/forgerock/android/auth/AppAuthConfigurer.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2020 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package org.forgerock.android.auth;
+
+import android.net.Uri;
+
+import androidx.annotation.NonNull;
+import androidx.browser.customtabs.CustomTabsIntent;
+import androidx.core.util.Consumer;
+
+import net.openid.appauth.AppAuthConfiguration;
+import net.openid.appauth.AuthorizationRequest;
+import net.openid.appauth.AuthorizationServiceConfiguration;
+
+import java.net.MalformedURLException;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * AppAuth Integration, https://github.com/openid/AppAuth-Android
+ * this class provides AppAuth integration and customization
+ */
+@RequiredArgsConstructor
+public class AppAuthConfigurer {
```

Comment:
Can this be package private?

Reply:
This is a public interface for developers to override the appAuth default configuration.